tests/namespace/test_mibs.py

```diff
@@ -36,6 +36,17 @@ def test_init_namespace_sync_d_lag_tables(self):
         self.assertTrue("PortChannel_Temp" in lag_name_if_name_map)
         self.assertTrue(lag_name_if_name_map["PortChannel_Temp"] == [])
 
+    def test_init_sync_d_interface_tables_for_recirc_ports(self):
```

Comment:
thank you, will be good to add test case to verify the oid output like in test_interfaces/test_pfc/test_queues

Reply:
For ex:
Interfaces MIB with this change:
iso.3.6.1.2.1.2.2.1.2.177 = STRING: "Ethernet6/45/1"
..
iso.3.6.1.2.1.2.2.1.2.10000 = STRING: "Management1/1"
**iso.3.6.1.2.1.2.2.1.2.11000 = STRING: "Recirc6/0/1"
iso.3.6.1.2.1.2.2.1.2.12000 = STRING: "Recirc6/0/0"**

Reply:
@SuvarnaMeenakshi extended test to verify the oids of recirc ports.